demo/meeting/package.json

```diff
@@ -8,7 +8,7 @@
     "@types/webpack-env": "^1.15.1",
     "@typescript-eslint/eslint-plugin": "^2.19.2",
     "@typescript-eslint/parser": "^2.19.2",
-    "amazon-chime-sdk-js": "^2.4.x",
+    "amazon-chime-sdk-component-library-react": "file:../../amazon-chime-sdk-component-library-react-1.6.0.tgz",
```

Comment:
Shouldn't this be 1.6.1, at the very least?

But I think you should look at `yalc`.

https://www.npmjs.com/package/yalc

Reply:
Update to yalc